docs/README.md

```diff
@@ -395,28 +395,28 @@ If you prefer to specify each parameter without following the instructions, you
 
 | Flag                   | Short version | Description |
 | :--------------------- | :------------ | :---------- |
-| `--homepage`           | `-H`          | the website of this project |
-| `--author`             | `-a`          | author of this project |
-| `--description`        | `-d`          | a short description |
-| `--license`            | `-l`          | license used in this project |
-| `--providerPackageName`| `-p`          | package name implementing the cloud provider integration where the application will be deployed |
-| `--repository`         | `-r`          | the URL of the repository |
-| `--version`            | `-v`          | the initial version |
+| `--homepage`           | `-H`          | The website of this project |
+| `--author`             | `-a`          | Author of this project |
+| `--description`        | `-d`          | A short description |
+| `--license`            | `-l`          | License used in this project |
+| `--providerPackageName`| `-p`          | Package name implementing the cloud provider integration where the application will be deployed |
+| `--repository`         | `-r`          | The URL of the repository |
+| `--version`            | `-v`          | The initial version |
 
-Additionally, you can use the flag `--skipInstall` if you want to skip installing dependencies and the flag `--skipGit ` in case you want to skip git initialization.
+Additionally, you can use the `--skipInstall` flag if you want to skip installing dependencies and the `--skipGit` flag in case you want to skip git initialization.
 
 > Booster CLI commands follow this structure: `boost <subcommand> [<flags>] [<parameters>]`.
 > Let's break down the command we have just executed:
 >
 > - `boost` is the Booster CLI executable
 > - `new:project` is the "subcommand" part. In this case, it is composed of two parts separated by a colon.
->   The first part, `new`, means that we want to generate a new resource. The second part, `project`, indicates which
->   kind of resource we are interested in. Other examples are `new:command`, `new:event`, etc. We'll see a bunch of them later.
+    >   The first part, `new`, means that we want to generate a new resource. The second part, `project`, indicates which
+    >   kind of resource we are interested in. Other examples are `new:command`, `new:event`, etc. We'll see a bunch of them later.
```

Comment:
It looks like there's a small indentation issue here that makes these greater than symbols render in the page:

<img width="913" alt="Screen Shot 2021-01-28 at 20 11 46" src="https://user-images.githubusercontent.com/175096/106193324-429cc300-61a5-11eb-9f83-a92aa2b1a54c.png">

Here's a one-click patch:

```suggestion
>   The first part, `new`, means that we want to generate a new resource. The second part, `project`, indicates which
>   kind of resource we are interested in. Other examples are `new:command`, `new:event`, etc. We'll see a bunch of them later.
```

Another option could be deleting the newlines to join lines 413, 414, and 412; they're rendered as a single paragraph anyway, but applying this patch should be fine too.